tools/java/org/unicode/cldr/api/FilteredData.java

```diff
@@ -0,0 +1,73 @@
+// © 2019 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+// © 2019 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+// © 2019 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+package org.unicode.cldr.api;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Filters underlying {@link CldrData} source data to transform values or remove values on the fly.
+ * The order of transformed values produced by the class is the same order as the underlying data.
+ *
+ * <p>This class does not currently support injecting additional values due to the difficulty in
+ * ensuring correct ordering semantics.
+ *
+ * <p>A subclass should implement the {@link FilteredData#filter(CldrValue)} method to achieve the
+ * desired results. Note however that this API is currently unstable and is expected to change once
```

Comment:
Does the CLDR project have/use doc tags like @draft and/or @provisional?
Since CLDR didn't have "API" before, this might be a good time to ask the TC how they want to mark unstable API.
Or is it too early to have a policy?

Reply:
It's worth thinking about, but I don't know of anything (and I suspect there is nothing).
We can easily retrofit any annotations for that (since nobody but ICU is going to be using this for a while).